### PR TITLE
add make test for run ut easily

### PR DESCRIPTION
**Pull request by LiZhenCheng9527**
**What type of PR is this?**
/kind enhancement

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by LiZhenCheng9527** [comment]
I'll add the documentation for running ut as soon as I can!

File: hack/run-ut.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+ROOT_DIR=$PWD
```
Comment (hzxuzhonghu) [inline hack/run-ut.sh:3]: pwd?

Comment (LiZhenCheng9527) [reply hack/run-ut.sh]: fixed

**Review by hzxuzhonghu — APPROVED** [review]

**Comment by hzxuzhonghu** [comment]
/lgtm

We should improve the scripts under `./hack`, not sure it they can run sucessfully because we use relative patch `./pkg`

**Pull request merged**